projects/portal/src/app/views/tools/app-tool/app-tool.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'view-app-tool',
+  templateUrl: './app-tool.component.html',
+  styleUrls: ['./app-tool.component.css'],
+})
+export class AppToolComponent implements OnInit {
+  apps: { name: string; link: string; icon: string }[];
+
+  constructor() {
+    this.apps = [
+      { name: 'Utilities', link: '/', icon: 'account_balance_wallet' },
```

Comment:
`assistant` is better for icon

https://fonts.google.com/icons?selected=Material+Symbols+Outlined:assistant:FILL@0;wght@400;GRAD@0;opsz@48